boards/kernel_layout.ld

```diff
@@ -0,0 +1 @@
+../chips/kernel_layout.ld
```

Comment:
If nothing else, it seems worthwhile to just get rid of this layer of indirection. I don't think we need this to be in the chips directory at all, can just have the raw file here.